perses/app/setup_relative_calculation.py

```diff
@@ -559,9 +570,12 @@ def run_setup(setup_options, serialize_systems=True, build_samplers=True):
             _forward_added_valence_energy = top_prop['%s_added_valence_energy' % phase]
             _reverse_subtracted_valence_energy = top_prop['%s_subtracted_valence_energy' % phase]
 
-            zero_state_error, one_state_error = validate_endstate_energies(_top_prop, _htf, _forward_added_valence_energy, _reverse_subtracted_valence_energy, beta = 1.0/(kB*temperature), ENERGY_THRESHOLD = ENERGY_THRESHOLD)#, trajectory_directory=f'{xml_directory}{phase}')
-            _logger.info(f"\t\terror in zero state: {zero_state_error}")
-            _logger.info(f"\t\terror in one state: {one_state_error}")
+            if not setup_options['use_given_geometries']:
```

Comment:
This won't work, and causes this error:
```
======================================================================
ERROR: perses.tests.test_fah.test_pipeline_small_molecule_solvent
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/test/lib/python3.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/runner/work/perses/perses/perses/tests/test_fah.py", line 71, in test_pipeline_small_molecule_solvent
    num_equilibration_steps_per_iteration = DEFAULT_EQ_STEPS_PER_ITERATION)
  File "/home/runner/work/perses/perses/perses/app/fah_generator.py", line 359, in run_neq_fah_setup
    setup_dict = run_setup(setup_options, serialize_systems=False, build_samplers=False)
  File "/home/runner/work/perses/perses/perses/app/setup_relative_calculation.py", line 573, in run_setup
    if not setup_options['use_given_geometries']:
KeyError: 'use_given_geometries'
-------------------- >> begin captured stdout << ---------------------
```
Instead, you want to check if the key is present first:
```python
if not 'use_given_geometries' in setup_options:
```